wifi-ap-sta

```diff
@@ -42,6 +43,23 @@ initialise_ap_values() {
 	HOSTAPD_country_code="${HOSTAPD_country_code:-$(wifi_country)}"
 }
 
+produce_ssid() {
+	local SSID;
+	if [ -f "/proc/cpuinfo" ]; then
+		SSID=$(grep "Serial" /proc/cpuinfo | awk '{print $3}')
+	fi
+	SSID="${SSID:-1234}"
+	SSID=$(echo "${SSID: -4}" | tr '[:lower:]' '[:upper:]')
+	echo "$(hostname)-${SSID}"
+}
+
+produce_password() {
+	local PASSPHRASE;
+	PASSPHRASE="$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c8)"
```

Comment:
this includes letters as well as _ and -
I think sticking with numbers is good for convenience - there's less potential for confusion over similar characters
```suggestion
	PASSPHRASE="$(< /dev/urandom tr -dc 0-9 | head -c8)"
```

Reply:
if you feel it's worth including letters for security, at least leave out the punctuation 'A-Za-z0-9'